vendor/vendor-android/base/src/main/kotlin/com/malinskiy/marathon/android/executor/listeners/CodeCoverageListener.kt

```diff
@@ -0,0 +1,29 @@
+package com.malinskiy.marathon.android.executor.listeners
+
+import com.malinskiy.marathon.android.AndroidDevice
+import com.malinskiy.marathon.device.DevicePoolId
+import com.malinskiy.marathon.device.toDeviceInfo
+import com.malinskiy.marathon.io.FileManager
+import com.malinskiy.marathon.io.FileType
+import com.malinskiy.marathon.log.MarathonLogging
+import com.malinskiy.marathon.test.TestBatch
+
+class CodeCoverageListener(
+    private val device: AndroidDevice,
+    private val devicePoolId: DevicePoolId,
+    private val fileManager: FileManager,
+    private val testBatch: TestBatch
+) : NoOpTestRunListener() {
+    private val logger = MarathonLogging.logger("CodeCoverageListener")
+
+    override fun testRunEnded(elapsedTime: Long, runMetrics: Map<String, String>) {
+        val src = "/sdcard/coverage-${testBatch.id}.ec"
```

Comment:
is it possible to obtain this part using device.getExternalStorageMount()?

Reply:
Yep, that works as well. I'll amend 👍